### DISCUSSION TITLE
some issues I have

### DISCUSSION THREAD

**Discussion by webodan** [rern/rAudio#486] [Bug Reports]
Hi, convert from Volumio here!

Pretty amazed with rAudio's Bluetooth functionality. Finally no more relying on rooted phones to play stuff from my phone!

However, I've got some issues I'd like to report:

1) mpdscribble won't scrobble from my web radios or my audio CDs using a PC DVD-ROM drive connected to an IDE to USB adapter. The scrobbler in Volumio did work with both of these, so it's an issue with mpdscribble I assume. MP3 files scrobble fine though.

2) I wasn't able to change my IP address from the default that the DHCP server gave to rAudio. I can deal with this though. (Edit. after a couple reboots it finally did change, so maybe it's a bit flaky idk).

3) I also couldn't use the rAudio remote control app because it didn't detect my Pi 3b+ running it. Would be nice if I could set up the IP address manually since I've got a funky setup on my network that seems to not resolve IPs on its own. (Solved by @cmh714 thanks!)

4) CPU usage shoots up to 100% when playing VGM files (megadrive/genesis chiptune files) off my USB hard drive, and playback starts stuttering and running very slow after 1 second of proper playback.  Tried rebooting rAudio but it still does this. It also for some reason changed from my USB DAC to the internal pi3 soundcard after rebooting. Switched back to it and now it sounds really loud even if i turn down the volume. The DAC is a PCM2704, in case it helps. Trying out SNES and GB chiptunes, as well as Amiga MOD and XM files works flawlessly, seems to only crap out with VGM/VGZ MegaDrive chiptune files.

Thanks again for this amazing work! Volumio gave me all sorts of problems. These are nothing compared to it.

**Comment by cmh714** [comment]
For the remote control app, first @rern doesnt make that one, regardless you can enter the IP to solve your problem.....swipe from left to right and you should get the pop-out where you can manually add the ip in the Support and Credits section. GL!

**Reply by webodan** [reply]
Thank you @cmh714 ! that was cryptic lol. Got it working now!

**Reply by cmh714** [reply]
Yeah its kind of hidden....glad to help!
